catalog-rest-service/src/main/resources/json/schema/metadataIngestion/workflow.json

```diff
@@ -128,7 +128,7 @@
       "properties": {
         "loggerLevel": {
           "type": "string",
-          "enum": ["DEBUG", "INFO", "WARN"],
+          "enum": ["DEBUG", "INFO", "WARN", "ERROR"],
```

Comment:
This is not needed here anymore. Being handled in https://github.com/open-metadata/OpenMetadata/pull/4564

Reply:
oh okay - reverting this